### PR TITLE
fix: bump axelar-core version to v1.0.2

### PR DESCRIPTION
**Pull request by ffe9f8**

### CODE REVIEW

**Review by StephenFluin — CHANGES_REQUESTED** [review]
`vald` is the same binary and should be updated to the same version

File: src/pages/resources/contract-addresses/mainnet.mdx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import UpgradePath from '/src/upgrade-path.md'
 | Variable              | Value     |
 | --------------------- | --------- |
 | `chain-id`            | `axelar-dojo-1` |
-| `axelar-core` version | `v0.35.5` |
+| `axelar-core` version | `v1.0.2` |
 | `vald` version        | `v0.35.5` |
```
Comment (milapsheth) [inline src/pages/resources/contract-addresses/mainnet.mdx:15]: Vald should be changed

**Comment by StephenFluin** [comment]
Check for other outdated references like this one: https://docs.axelar.dev/dev/amplifier/chain-integration/integrate-a-chain/

File: src/pages/validator/setup/manual.mdx [hunk 2]
```diff
@@ -16,16 +16,16 @@ Check the appropriate version for the network accordingly:
 
 <tabs>
 <tab-item title="Mainnet">
- ```bash
- AXELARD_RELEASE=v0.35.5
+```bash
+AXELARD_RELEASE=v0.35.5
```
Comment (StephenFluin) [inline src/pages/validator/setup/manual.mdx:20]: This should be 1.0.2 as well.

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Still missing pages like https://docs.axelar.dev/dev/amplifier/chain-integration/integrate-a-chain/

**Review by StephenFluin — CHANGES_REQUESTED** [review]
This now specifies an incorrect version of tofnd.

I believe tofnd is still 1.0.1.

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**